bika/lims/browser/analysisrequest/add2.py

```diff
@@ -349,12 +367,16 @@ def generate_fieldvalues(self, count=1):
         # generate fields for all requested ARs
         for arnum in range(count):
             source = copy_from.get(arnum)
+            parent = None
+            if source is not None:
+                parent = self.get_parent_ar(source)
             for field in fields:
                 value = None
                 fieldname = field.getName()
                 if source and fieldname not in SKIP_FIELD_ON_COPY:
                     # get the field value stored on the source
-                    value = self.get_field_value(field, source)
+                    context = parent or source
+                    value = self.get_field_value(field, context)
```

Comment:
I still don't know why I have to fetch the "root AR" (`-R01`) to have the Client and Profiles field filled with the values. At least if feels like this is some kind of workaround.